src/app/ui-kit/typography/typography.component.html

```diff
@@ -0,0 +1,3 @@
+<p [ngClass]="[classes]" >
```

Comment:
p tag should only be applied to paragraph (P1, P2, Label in your case). So h1-h6 tags should be used for H1-H6